### PR TITLE
Otter Grade Dockerfile Added wget

### PR DESCRIPTION
**Pull request by sean-morris**
- wget added to Dockerfile for backwards compatibility with autograder.zip files generated with otter-grader version 4

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 **v5.6.0(unreleased):**
 
 * final_grades.csv: notebook name is the file name for each row instead of the full path to the notebook and the rows are sorted by notebook name
+* added wget to grade/Dockerfile -- this allows otter-grader version 4 generated autograder.zips to work with otter-grader version 5
```
Comment (chrispyles) [inline CHANGELOG.md:6]: generally I prefer to leave implementation details out of the changelog since it has a wide audience; can we reword this to something like "Added backwards compatibility to Otter Grade for autograder configuration zip files generated in previous major versions of Otter-grader"

**Review by chrispyles — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sean-morris) [reply CHANGELOG.md]: Cool. Sent it through. Thanks!

**Review by chrispyles — APPROVED** [review]
